Author: Sergei Golovan <sgolovan@debian.org>
Description: Patch defines HiPE architecture ppc for powerpc arch.
Last-Modified: Mon, 08 Mar 2010 12:23:16 +0300

--- a/erts/configure.in
+++ b/erts/configure.in
@@ -612,6 +612,7 @@
     x86_64)	ARCH=amd64;;
     amd64)	ARCH=amd64;;
     macppc)	ARCH=ppc;;
+    powerpc)	ARCH=ppc;;
     ppc)	ARCH=ppc;;
     ppc64)	ARCH=ppc64;;
     "Power Macintosh")	ARCH=ppc;;
